packages/squiggle-lang/src/public/SqProject/SqModuleOutput.ts

```diff
@@ -168,7 +168,18 @@ export class SqModuleOutput {
     };
 
     const started = new Date();
-    const runResult = await params.runner.run(runParams);
+    let runResult;
+    try {
+      runResult = await params.runner.run(runParams);
```

Comment:
This part and changes in `runners/*` is unrelated to the fix, but it helped me to debug the error, which wasn't marshaled from the worker properly.